src/core/verbs/global-verbs-sanitizer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { sanitizeVerbSelectionList } from './gobal-verbs-sanitizer';
+
+describe('sanitizeVerbSelectionList', () => {
+  it('should return empty array when it feeds sanitizeVerbSelectionList equals undefined', () => {
+    // Arrange
+    const verbListOne: string[] = undefined;
+    const verbListTwo: string[] = undefined;
+
+    // Act
+    const result: string[] = sanitizeVerbSelectionList(
+      verbListOne,
+      verbListTwo
+    );
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+  it('should return empty array when it feeds sanitizeVerbSelectionList equals null', () => {
+    // Arrange
+    const verbListOne: string[] = null;
+    const verbListTwo: string[] = null;
+
+    // Act
+    const result: string[] = sanitizeVerbSelectionList(
+      verbListOne,
+      verbListTwo
+    );
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+  it('should return empty array when it feeds sanitizeVerbSelectionList equals empty array', () => {
+    // Arrange
+    const verbListOne: string[] = [];
+    const verbListTwo: string[] = [];
+
+    // Act
+    const result: string[] = sanitizeVerbSelectionList(
+      verbListOne,
+      verbListTwo
+    );
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+  it('should return one item when it feeds sanitizeVerbSelectionList with two items', () => {
```

Comment:
There are more cases missing, e.g.:

1. None of the verbs are in the list should return empty array
2. Some of the verbs are in the list but not the first (ej 3) ones in the array.
3. Some of the verbs are in the list but the last (ej 3) in the array
4. Some of the verbs are in the list but they are mixed in the array (Some of the are some of them not)